logger/logger.go

```diff
@@ -16,3 +17,7 @@ type Interface interface {
 }
 
 var Default = dlog.New(log.Default().Writer(), "", log.LstdFlags, os.Getenv("DEBUG") == "1")
+
+// note: previously ioutil.Discard which is not deprecated in favord of io.Discard
+// so this is valid only from go1.16
+var Silent = dlog.New(io.Discard, "", log.LstdFlags, os.Getenv("DEBUG") == "1")
```

Comment:
Thank you for the PR!

There are generics in this project as well, I doubt that anyone would successfully compile this on Go prior to 1.18 😂